### PR TITLE
Add and use AP_WINCH_ENABLED

### PR DESCRIPTION
**Pull request by peterbarker**
```
Board,AP_Periph,blimp,bootloader,copter,heli,iofirmware,plane,rover,sub
Durandal,,*,*,*,*,,*,*,*
Hitec-Airspeed,0,,,,,,,,
KakuteH7-bdshot,,*,*,*,*,,*,*,*
MatekF405,,*,*,*,*,,*,*,*
Pixhawk1-1M,,8,*,8,8,,8,8,8
f103-QiotekPeriph,0,,,,,,,,
f303-Universal,*,,,,,,,,
iomcu,,,,,,*,,,
revo-mini,,*,*,*,*,,*,*,*
skyviper-v2450,,,,*,,,,,
```

`Pixhaw1-1M` gets more stuff in its hwdef.h from `minimize.inc`

### CODE REVIEW

File: ArduCopter/config.h [hunk 1]
```diff
@@ -147,7 +147,7 @@
 //////////////////////////////////////////////////////////////////////////////
 // winch support
 #ifndef WINCH_ENABLED
```
Comment (rmackay9) [inline ArduCopter/config.h:149]: maybe we can also replace WINCH_ENABLED with AP_WINCH_ENABLED?

Comment (peterbarker) [reply ArduCopter/config.h]: Hmmm.  For some reason I wasn't game to do that.

I've pushed up another commit which does it - and the results from build it are the same as above (i.e. no compile-time changes apart from minimized boards which get another line in their hwdef

Comment (peterbarker) [reply ArduCopter/config.h]: (oh and SkyViper gets a little more in its hwdef too)

**Review by rmackay9 — APPROVED** [review]
as long as its been tested with a couple of compile tests it should be fine, thanks!

No need to add this to the custom build server?  .. maybe by some magic it will appear automatically?

**Comment by peterbarker** [comment]
> as long as its been tested with a couple of compile tests it should be fine, thanks!
> 
> No need to add this to the custom build server? .. maybe by some magic it will appear automatically?

No, it would need to be added :-)  That would be some deep, dark juju...

**Pull request merged**